docs/installation.md

```diff
@@ -75,8 +75,11 @@ In order to start the installation process, you must boot your computer from the
 * Press <kbd>F12</kbd> (or the appropriate key) and select the install drive&mdash;usually "USB-HDD" or something containing the word "USB", but the wording may vary. If you choose the incorrect drive, your computer will likely continue to boot as normal. Just restart your computer and pick a different drive in that menu.
 * Shortly after selecting the appropriate boot drive, you should be presented with the elementary OS splash screen. You may now follow the on-screen instructions which will guide you through the rest of the process.
 
-</div>
+## Default LiveCD login credentials
+
+* Enter <kbd>elementary</kbd> as the username and leave the password field blank in order to log into the LiveCD instance
 
+</div>
```

Comment:
I'm pretty sure the Live CD should never lock, or at least isn't intended to, although I have had several reports of it doing so. As such, I cannot confirm that these are the right credentials.

Secondly, I believe we would want this outside of the source-os specific switcher rather than just being in the Windows area.